routers/api/v1/repo/pull.go

```diff
@@ -527,7 +527,7 @@ func EditPullRequest(ctx *context.APIContext) {
 	// Pass one or more user logins to replace the set of assignees on this Issue.
 	// Send an empty array ([]) to clear all assignees from the Issue.
 
-	if ctx.Repo.CanWrite(unit.TypePullRequests) && (form.Assignees != nil || len(form.Assignee) > 0) {
+	if ctx.Repo.CanWrite(unit.TypePullRequests) {
```

Comment:
IMO the code worked like:

* form.Assignees = nil: no update
* form.Assignees = []: Send an empty array ([]) to clear all assignees from the Issue. (the comment above)
* form.Assignees = [...]: update assignees

Is this PR's change fine?

Reply:
My bad... the issue attributes were not loaded.